### ISSUE TITLE
Can't run starter project

### ISSUE THREAD

**Issue by keesvanbemmel** [becem-gharbi/directus-starter#1]
Hi! I can't run your starter project. I suspect it is an issue in your naiveui module, but unsure.

When I run `pnpm dev` I get this stack:
```
 ERROR  Cannot read properties of undefined (reading 'push')                                                                                                         1:30:49 PM

  at prepare:types (node_modules/.pnpm/@bg-dev+nuxt-naiveui@1.2.6/node_modules/@bg-dev/nuxt-naiveui/dist/module.mjs:27:38)
  at node_modules/.pnpm/hookable@5.5.3/node_modules/hookable/dist/index.mjs:48:66
  at Object.run (node_modules/.pnpm/hookable@5.5.3/node_modules/hookable/dist/index.mjs:41:43)
  at node_modules/.pnpm/hookable@5.5.3/node_modules/hookable/dist/index.mjs:48:56
  at async writeTypes (node_modules/.pnpm/@nuxt+kit@3.7.0/node_modules/@nuxt/kit/dist/index.mjs:2872:3)
  at async Object.run (node_modules/.pnpm/nuxi@3.7.0/node_modules/nuxi/dist/chunks/prepare.mjs:70:5)
  at async runCommand$1 (node_modules/.pnpm/nuxi@3.7.0/node_modules/nuxi/dist/shared/nuxi.51fb54b9.mjs:1644:5)
  at async runCommand$1 (node_modules/.pnpm/nuxi@3.7.0/node_modules/nuxi/dist/shared/nuxi.51fb54b9.mjs:1638:7)
  at async runMain$1 (node_modules/.pnpm/nuxi@3.7.0/node_modules/nuxi/dist/shared/nuxi.51fb54b9.mjs:1756:7) 



 ERROR  Cannot read properties of undefined (reading 'push')                                                                                                         1:30:49 PM


 ERROR  Command failed with exit code 1.
```

**Comment by becem-gharbi** [comment]
Hi, what is the nuxt version you're using?

**Comment by keesvanbemmel** [comment]
wow you're quick.

I cloned your repo and ran `pnpm dev` that showed Nuxt 3.7.0.

**Comment by becem-gharbi** [comment]
Nuxt v3.7 is released today. You can try with v3.6 at the moment.

**Comment by keesvanbemmel** [comment]
Solves it! Thanks!

**Comment by becem-gharbi** [comment]
This issue is fixed on [v1.2.7](https://github.com/becem-gharbi/nuxt-naiveui/releases/tag/v1.2.7).
